detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Project/ProjectXmlResolver.cs

```diff
@@ -82,26 +93,67 @@ public DependencyResult Process()
                     {
                         XmlAttribute include = attributes["Include"];
                         XmlAttribute version = attributes["Version"];
+                        XmlAttribute versionOverride = attributes["VersionOverride"];
                         String versionStr = null;
-                        if (version == null)
+                        String versionOverrideStr = null;
+                        
+                        if (versionOverride == null && version == null)
                         {
                             foreach (XmlNode node in package.ChildNodes)
                             {
+                                if (node.Name == "VersionOverride")
+                                {
+                                    versionOverrideStr = node.InnerXml;
+                                    break;
+                                }
                                 if (node.Name == "Version")
                                 {
                                     versionStr = node.InnerXml;
-                                    break;
                                 }
                             }
                         }
+                        else if (versionOverride != null)
+                        {
+                            versionOverrideStr = versionOverride.Value;
+                        }
                         else
                         {
                             versionStr = version.Value;
                         }
-                        if (include != null && !String.IsNullOrWhiteSpace(versionStr))
+
+                        if (include != null)
```

Comment:
This whole logic is needed to check that we are adding the dependency correctly from props file and also because we do not want to hinder the old logic we had for PackageReferences.